### ISSUE TITLE
Error: spawn ENOENT

### ISSUE THREAD

**Issue by dKab** [SpookyJS/SpookyJS#206]
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:988:11)
    at Process.ChildProcess._handle.onexit (child_process.js:779:34)

when trying to run node examples/hello.js

**Comment by natsu90** [comment]
This is due to it can not find `casperjs` command. Do install `casperjs` globally, `npm install -g casperjs`, or add `child.command` property when initialize Spooky,

``` js
...
var spooky = new Spooky({
        child: {
            command: "./node_modules/.bin/casperjs", // add this line
            transport: 'http'
        },
        casper: {
            logLevel: 'debug',
            verbose: true
        }
    }, function (err) {
...
```

**Comment by dKab** [comment]
Thanks, I figured it already, although I didn't know about `command` option. Often installing packages globally is not an option - for instanse when you want to deploy your app on heroku.  Therefore I manually added `.bin` folder to PATH:

```
var env = _.clone(process.env),
        path = __dirname + '/node_modules/.bin';

    env.PATH += ':' + path;

    var spooky = new Spooky({
        child: {
            transport: 'http',
            spawnOptions: {
                env: env
            }
        },
```

Actually, if you don't have phantomJS installed globally (my case) the `command` option won't work, because casperJS won't find phantomJS executable.  I think the only thing that can help in this situation is this little trick with PATH.
